mocks/formulario-verification-correct.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Verified account</title>
```

Comment:
I'd suggest a title change.
Maybe something along the lines of "Account verification", refering to the process per se, instead of guaranteeing that the verification was successful.